staging/src/k8s.io/client-go/util/jsonpath/jsonpath.go

```diff
@@ -523,3 +525,11 @@ func (j *JSONPath) evalToText(v reflect.Value) ([]byte, error) {
 	fmt.Fprint(&buffer, iface)
 	return buffer.Bytes(), nil
 }
+
+// findJsonPath extracts a set of values from a runtime object using a parser
```

Comment:
I don't think this is desired here. I'm fine with duplicating that function in both places you need it. It's too small to bother making it a helper and this is definitely not a place for it.